examples/next/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@aws-amplify/ui-react": "^5.0.0",
-    "@aws-amplify/ui-react-liveness": "^1.0.1",
+    "@aws-amplify/ui-react-liveness": "^1.0.2",
```

Comment:
This still should install `^1.0.2-liveness-test` correct?

Reply:
fixed thanks!